sources/packages/web/src/types/contracts/institution/Application.ts

```diff
@@ -0,0 +1,12 @@
+export interface ApplicationDetails {
+  applicationProgramName: string;
```

Comment:
IMO, the object already name Application does not need to have all the properties prefixed with "application" again.
I would recommend removing it.

Reply:
mm, since these variables are used in the form. I gave `application` as a prefix, 
![image](https://user-images.githubusercontent.com/77353155/135360225-18af692e-ac77-444e-a0c2-3ab0cd78c5d5.png)